app/controllers/settings_controller.rb

```diff
@@ -0,0 +1,69 @@
+class SettingsController < ApplicationController
+  def index
+    @settings = Setting.find_by(company_id: current_company.id)
+    if @settings.present?
+      if(@settings.working_days.class == String)
+        @settings.working_days = JSON.parse(@settings.working_days.gsub("'",'"').gsub('=>',':'))
+        @settings.timings = JSON.parse(@settings.timings.gsub("'",'"').gsub('=>',':'))
+      end
+      @settings
+    end
+  end
+
+  def edit
+    @settings = Setting.find_by(company_id: current_company.id)
+    if @settings != nil
+      if(@settings.working_days.class == String)
+        @settings.working_days = JSON.parse(@settings.working_days.gsub("'",'"').gsub('=>',':'))
+        @settings.timings = JSON.parse(@settings.timings.gsub("'",'"').gsub('=>',':'))
+      end
+      @settings
+    end
+  end
+
+  def update
+    days = [:monday, :tuesday, :wednesday, :thursday, :friday, :saturday, :sunday]
+    on_days = {}
+    days.each do |day|
+      if params[day].blank?
```

Comment:
`on_days[day] = false if params[day].blank?
on_days[day] = true if !params[day].blank?`

isn't this more convinient

Reply:
isn't both are fine?